autoload/db_ui/drawer.vim

```diff
@@ -9,6 +9,11 @@ function db_ui#drawer#get()
   return s:drawer_instance
 endfunction
 
+" Default drawer sections and their order
+if !exists('g:db_ui_drawer_sections')
+  let g:db_ui_drawer_sections = ['new_query', 'buffers', 'saved_queries', 'schemas']
```

Comment:
Let's add the default in `plugin/db_ui.vim` where other defaults are. This should also fix the tests.